SourceS/sdl2_to_1_2_backports.h

```diff
@@ -244,6 +245,18 @@ inline void SDLBackport_PixelformatToMask(int pixelformat, Uint32 *flags, Uint32
 	}
 }
 
+// A limited implementation of `a.format` == `b.format` from SDL2.
+inline bool SDLBackport_PixelFormatFormatEq(const SDL_PixelFormat *a, const SDL_PixelFormat *b)
+{
+	return a->BitsPerPixel == b->BitsPerPixel && (a->palette != nullptr) == (b->palette != nullptr);

```

Comment:
why the parentheses?

Reply:
Just for clarity: We're checking that either they both have a palette, or neither of them do (presence of a palette in SDL1 corresponds to the `INDEXED` family of formats in SDL2 as far as I can tell)

Reply:
Oh I missed that you where comparing the two bools to each other afterwards.